plugins/fossa/src/components/FossaPage/FossaPage.tsx

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Entity,
+  EntityName,
+  RELATION_OWNED_BY,
+} from '@backstage/catalog-model';
+import {
+  Content,
+  Header,
+  Page,
+  StatusError,
+  StatusOK,
+  StatusWarning,
+  Table,
+  TableColumn,
+  TableFilter,
+  useApi,
+} from '@backstage/core';
+import {
+  catalogApiRef,
+  EntityRefLink,
+  EntityRefLinks,
+  formatEntityRefTitle,
+  getEntityRelations,
+} from '@backstage/plugin-catalog-react';
+import { Tooltip } from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
+import { DateTime } from 'luxon';
+import * as React from 'react';
+import { useMemo } from 'react';
+import { useAsync } from 'react-use';
+import { FindingSummary, fossaApiRef } from '../../api';
+import { useProjectNames } from '../useProjectNames';
+
+type FossaRow = {
+  entity: Entity;
+  resolved: {
+    name: string;
+    ownedByRelationsTitle?: string;
+    ownedByRelations: EntityName[];
+    loading: boolean;
+    details?: FindingSummary;
+  };
+};
+
+const columns: TableColumn<FossaRow>[] = [
+  {
+    title: 'Name',
+    field: 'resolved.name',
+    highlight: true,
+    render: ({ entity }) => <EntityRefLink entityRef={entity} />,
```

Comment:
Maybe defaultKind Component? Assume it'll be overwhelmingly the most common.

Sometimes I have been wondering if we could signal the kind better, somehow. Like, a newline and muted-color kind, or an icon .... or something

Reply:
Icon would be neat, like it's already done in the catalog import. But maybe not something for this PR.